Python/thread_pthread.h

```diff
@@ -142,9 +142,19 @@ typedef struct {
     pthread_mutex_t  mut;
 } pthread_lock;
 
+#define PUTS(fd, str) _Py_write_noraise(fd, str, (int)strlen(str))
+
 #define CHECK_STATUS(name)  if (status != 0) { perror(name); error = 1; }
-#define CHECK_STATUS_PTHREAD(name)  if (status != 0) { fprintf(stderr, \
-    "%s: %s\n", name, strerror(status)); error = 1; }
+/* CHECK_STATUS_PTHREAD is async-signal-safe */
+#define CHECK_STATUS_PTHREAD(name) \
```

Comment:
I don't know. I'm inclined not to remove them right now.
OTOH, this whole PR may be made obsolete by https://github.com/python/cpython/pull/2408.